Rakefile

```diff
@@ -89,6 +85,6 @@ task :run do
       next
     end
 
-    HTTP.post(ENV.fetch("BADGER_SLACK_WEBHOOK_URL"), body: JSON.dump(message_payload))
+    HTTP.post(ENV.fetch("BADGER_SLACK_WEBHOOK_URL"), body: JSON.dump(message_payload)) unless message.nil?
```

Comment:
To make this a little easier to follow, I'd move the `nil` check onto the `message_payload` hash itself, i.e.

```ruby
message_payload = nil
if messages.any?
  message_payload = {
    ...
  }
end

HTTP.post(...) unless message_payload.nil?
```

Reply:
I've changed it now